tslint.json

```diff
@@ -148,19 +197,53 @@
             "check-whitespace"
         ],
         "one-variable-per-declaration": true,
-        "only-arrow-functions": false,
-        "ordered-imports": false,
+        "only-arrow-functions": [true, "allow-declarations", "allow-named-functions"],
+        "ordered-imports": [
+            false
+        ],
+        "possible-timing-attack": true,
         "prefer-array-literal": false,
         "prefer-const": true,
+        "prefer-for-of": false,
+        "prefer-function-over-method": false,
+        "prefer-method-signature": false,
+        "prefer-template": false,
         "prefer-type-cast": true,
+        "strict-type-predicates": false,
+        "promise-function-async": false,
         "promise-must-complete": true,
         "quotemark": [
             true,
             "single"
         ],
         "radix": false,
+        "react-a11y-anchors": true,
```

Comment:
Yes, per PR description above, fetched a fresh copy of [microsoft-tslint-contrib/tslint.json](https://github.com/Microsoft/tslint-microsoft-contrib/blob/master/tslint.json) and edited the settings to match our previous values (see Jira for list of edits) and then edited our code to address any linting errors.